packages/commonwealth/client/scripts/stores/ActivityStore.ts

```diff
@@ -32,12 +32,12 @@ export class ActiveThreadsStore {
     const reactions = {};
     const allThreads = this.getThreadsByCommunity(parentEntity);
     allThreads.forEach((thread) => {
-      const allComments = app.comments.getByThread(thread);
-      comments[thread.id] = allComments;
-      reactions[thread.id] = app.comments.getReactionByPost(thread);
-      allComments.forEach((c) => {
-        reactions[thread.id].concat(app.comments.getReactionByPost(c));
-      });
+      // const allComments = app.threads.getByThread(thread);
+      // comments[thread.id] = allComments;
+      // reactions[thread.id] = app.threads.getReactionByPost(thread);
+      // allComments.forEach((c) => {
+      //   reactions[thread.id].concat(app.threads.getReactionByPost(c));
+      // });
```

Comment:
I think we can clear-up a big portion of ActivityStore as most functions are not used anywhere

Reply:
🚀